bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -70,17 +70,15 @@ else if (!userIsRegistered(userService.getCurrentUser().getUserId())
       if (request.getRequestURI().endsWith("profile.jsp")
           || request.getRequestURI().endsWith("register.jsp")
           || request.getRequestURI().endsWith("login.jsp")) {
-        RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/profile.jsp");
-        requestDispatcher.forward(request, response);
-        return;
-        //Case: User is trying to access ADMIN features of the application
-      } else if (request.getRequestURI().endsWith("requests.html")) {
-          if (getCurrentUserPermission() == Permission.ADMIN) {
-              chain.doFilter(req, res);
-          } else {
-            RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/profile.jsp");
-            requestDispatcher.forward(request, response);  
-          }
+              if (getCurrentUserPermission()==Permission.USER) {
```

Comment:
Formatting: add spacing around the double equals.

Reply:
Added spacing